src/fiatconnect/slice.ts

```diff
@@ -16,30 +17,56 @@ export interface State {
   quotes: (FiatConnectQuoteSuccess | FiatConnectQuoteError)[]
   quotesLoading: boolean
   quotesError: string | null
+  fiatAccount: ObfuscatedFiatAccountData | null
+  fiatAccountLoading: boolean
+  fiatAccountError: string | null
+  mostRecentFiatAccountIds: FiatAccount[]
```

Comment:
from the name I'd expect this to just contain fiat account id's, not other info about fiat accounts. also why "most recent" if it's a list? is there an eviction policy?

Reply:
nvm seeing that now